srcpkgs/sublime-text4/template

```diff
@@ -0,0 +1,41 @@
+# Template file for 'sublime-text4'
+pkgname=sublime-text4
+version=4107
+revision=1
+depends="libpng gtk+ hicolor-icon-theme desktop-file-utils"
+short_desc="Sophisticated text editor for code, markup and prose"
+maintainer="DirectorX <void.directorx@protonmail.com>"
+license="custom:Proprietary"
+homepage="https://www.sublimetext.com/"
+conflicts="sublime-text3"
+
+
+case "$XBPS_TARGET_MACHINE" in
+		x86_64)
+		distfiles="https://download.sublimetext.com/sublime_text_build_${version}_x64.tar.xz"
+		checksum=c7d9e7520ea91c2db6c026d1f48e9067457d55158393be042100a1161a3e5915
+		;;
+		aarch64)
+		distfiles="https://download.sublimetext.com/sublime_text_build_${version}_arm64.tar.xz"
+		checksum=e0ccf0859d7b73a718b65c0f03e48189ecba1a256bb800d738b24078994cae03
+		;;
+esac
+
+repository="nonfree"
+archs="x86_64 aarch64"
+wrksrc="sublime_text"
+nopie=yes
+
+do_install() {
+	vmkdir "usr/bin"
+	vmkdir "usr/lib/sublime_text"
+	cp --preserve=mode -r ./* "${DESTDIR}/usr/lib/sublime_text"
+	vinstall "sublime_text.desktop" 644 "usr/share/applications/"
+	for size in 128 16 256 32 48; do
+		vinstall "Icon/${size}x${size}/sublime-text.png" 644 \
+		 "usr/share/icons/hicolor/${size}x${size}/apps/"
+	done
+	ln -s /usr/lib/sublime_text/sublime_text ${DESTDIR}/usr/bin/subl4
+	vsed -e 's:Exec=/opt/sublime_text/:Exec=/usr/lib/sublime_text/:' \
+	 -i ${DESTDIR}/usr/share/applications/sublime_text.desktop
```

Comment:
it would be nice to indent this line when it's a continuation of the previous one